src/index.html

```diff
@@ -2,13 +2,62 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        class="logo"
+      >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+            >Laptops & Computers</a>
```

Comment:
```suggestion
            >
              Laptops & Computers
            </a>
```
Consider fixing indents according to the code style guide